lib/utils/utils.js

```diff
@@ -224,18 +224,37 @@ function proposeAlternative(word, _dictionary, _exceptions) {
   return propose(word, dictionary, {threshold: 0.3});
 }
 
-function getExternalContractUrl(file) {
+function getExternalContractUrl(file,ipfsUrl) {
   const constants = require('../constants');
   let url;
   const RAW_URL = 'https://raw.githubusercontent.com/';
   const MALFORMED_ERROR = 'Malformed Github URL for ';
+  const MALFORMED_IPFS_ERROR = 'Malformed IPFS URL for ';
   if (file.startsWith('https://github')) {
     const match = file.match(/https:\/\/github\.[a-z]+\/(.*)/);
     if (!match) {
       console.error(MALFORMED_ERROR + file);
       return null;
     }
     url = `${RAW_URL}${match[1].replace('blob/', '')}`;
+  } else if (file.startsWith('ipfs')) {
```

Comment:
If ipfsUrl is unavailable, we should print an error along the lines of "IPFS getUrl is not available. Please set it in your storage config. For more info: https://embark.status.im/docs/storage_configuration.html"

Reply:
Ok, i will fix it tomorrow.thanks